cmd/flags_location.go

```diff
@@ -28,8 +28,12 @@ var (
 		Usage: "Address (URL form) of ipify service",
 		Value: "https://api.ipify.org/",
 	}
+	locationBuiltInDbFlag = cli.BoolTFlag{
```

Comment:
Dont need this flag, compiled-in should be default

Reply:
Removed